looper/looper.py

```diff
@@ -555,20 +566,700 @@ def __call__(self, args, remaining_args):
+                        # command_name = os.path.basename(str(glob.glob(os.path.join(
+                                            # self.prj.metadata.results_subdir,
+                                            # sample_name, '*commands.sh'))[0]))
+                    else:
+                        log_name = str(single_sample.iloc[0]['annotation']) + "_log.md"
+                        # Currently unused. Future?
+                        # profile_name = str(single_sample.iloc[0]['annotation']) + "_profile.tsv"
+                        # command_name = str(single_sample.iloc[0]['annotation']) + "_commands.sh"
+                    log_file = os.path.join(self.prj.metadata.results_subdir,
+                                            sample_name, log_name)
+                    log_relpath = os.path.relpath(log_file, reports_dir)
+                    if os.path.isfile(log_file):
+                        html_file.write(STATUS_ROW_LINK.format(
+                                            row_class="",
+                                            file_link=log_relpath,
+                                            link_name=log_name))
+                    else:
+                        # Leave cell empty
+                        html_file.write(STATUS_ROW_LINK.format(
+                                            row_class="",
+                                            file_link="",
+                                            link_name=""))
+                    # Fourth Col: Sample runtime (if completed)
+                    # If Completed, use stats.tsv
+                    stats_file = os.path.join(
+                                    self.prj.metadata.results_subdir,
+                                    sample_name, "stats.tsv")
+                    if os.path.isfile(stats_file):
+                        t = _pd.read_table(stats_file, header=None,
+                                           names=['key', 'value', 'pl'])
+                        t.drop_duplicates(subset=['key', 'pl'],
+                                          keep='last', inplace=True)
+                        try:
+                            time = str(t[t['key'] == 'Time'].iloc[0]['value'])
+                            html_file.write(STATUS_ROW_VALUE.format(
+                                            row_class="",
+                                            value=str(time)))
+                        except IndexError:
+                            warnings.append("The stats_summary.tsv file is incomplete")                        
+                    else:
+                        html_file.write(STATUS_ROW_VALUE.format(
+                                            row_class=button_class,
+                                            value="Unknown"))
+                    html_file.write(STATUS_ROW_FOOTER)
+
+                html_file.write(STATUS_FOOTER)
+                html_file.write(HTML_FOOTER)
+                html_file.close()
+                if warnings:
```

Comment:
What does this look like when printed? I'm probably missing something, but at first glance it seems odd to join messages on an empty string.

Reply:
should probably be a space...

Reply:
Addressed with 769841e.  It was supposed to be empty, because it should be a single unique message.  Updated to be a true/false check and then simply use the singular message.